### ISSUE TITLE
Magisk 11.6 not detected whenever a module requiring automount is installed

### ISSUE THREAD

**Issue by Zackptg5** [topjohnwu/Magisk#135]
Whenever I install a module that requires automount (and has stuff to mount), magisk manager doesn't detect magisk. I have magisk 11.6 installed on crdroid 2.4 for lg g2 (vs980). Here's a logcat:
[2017-03-23-20-21-21.txt](https://github.com/topjohnwu/Magisk/files/866543/2017-03-23-20-21-21.txt)
An error that stood out to me was: 
CANNOT LINK EXECUTABLE "/system/bin/sh": library "libsigchain.so" not found
and a whole bunch or these link errors spawn off of this because libsigchain.so isn't found. 
Here's magisk log: [magisk_error_20170323_203508.txt](https://github.com/topjohnwu/Magisk/files/866552/magisk_error_20170323_203508.txt)

The weird thing is that magisksu is still detected so magisk is clearly installed but something goofy is happening 
 Here's a screenshot from magisk manager: 
![screenshot_20170323-203618](https://cloud.githubusercontent.com/assets/5107713/24275798/7dc01e98-1008-11e7-9e3e-bcaeb73e0f4d.png)

**Comment by Zackptg5** [comment]
Update on the issue, I figured out the cause. It happens under a specific set of circumstances.
A module needs to create a new system folder (in this case: system/magisk.d) and have automount enabled as in this test module:
[Arise Magisk Compatibility Module Test.zip](https://github.com/topjohnwu/Magisk/files/866579/Arise.Magisk.Compatibility.Module.Test.zip)

Once I disabled automount for it, the problem went away.
This issue also occurred on my nexus 5x running screwd so it has nothing to do with rom dependencies
